wandb/sdk/wandb_run.py

```diff
@@ -3586,9 +3592,14 @@ def __getattr__(self, item: str) -> Any:
         self._assert_instance()
         return getattr(self._instance, item)
 
-    def wait(self) -> ArtifactInterface:
+    def wait(self, wait_timeout_secs: int = None) -> ArtifactInterface:
```

Comment:
Nit: could we name this `timeout` or `timeout_sec`? The `wait` seems redundant, and we can't really change this after release.